src/Builder/Payload/CapturePayloadBuilder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
```

Comment:
You need to add the license header (stuff like `* @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0` ...) 😉

Reply:
There a composer script `set-license-header` ;)